task/net/src/main.rs

```diff
@@ -241,7 +245,7 @@ fn main() -> ! {
                     Timers::Watchdog => panic!("MAC RX watchdog"),
                 }
             }
-            let mut msgbuf = [0u8; ServerImpl::INCOMING_SIZE];
+            let mut msgbuf = [0u8; ServerImpl::<BspImpl>::INCOMING_SIZE];
```

Comment:
We could use `idl::INCOMING_SIZE` here, then remove `INCOMING_SIZE` from the `ServerImpl` entirely.